Task50/Google/SeleniumEasy/Person.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Task50Pages.SeleniumEasy;
+
+internal class Person
+{
+    string Name;
```

Comment:
Please, add access modifiers here and use "get" and "set" words to transform these fields to properties